src/Filter/FilterComposite.php

```diff
@@ -35,6 +36,13 @@ class FilterComposite implements FilterInterface
     const CONDITION_AND = 2;
 
     /**
+     * Error message for non callable variables
+     */
+    const ERROR_MESSAGE = <<<EOT
+The value of %s should be either a callable or an instance of Zend\Stdlib\Hydrator\Filter\FilterInterface
```

Comment:
This smell bad, if same error message its used on different places then means its not enough informative about what is the source of the problem.